airbyte-integrations/connectors/source-oracle-v2/metadata.yaml

```diff
@@ -0,0 +1,28 @@
+data:
+  ab_internal:
+    ql: 200
+    sl: 100
+  allowedHosts:
+    hosts:
+      - ${host}
+      - ${tunnel_method.tunnel_host}
+  connectorSubtype: database
+  connectorType: source
+  definitionId: b39a7370-74c3-45a6-ac3a-380d48520a83
+  dockerImageTag: 0.0.1
```

Comment:
Since you are using the ID of the existing connector (which will make rolling this out much easier), I think the dockerImageTag needs to be greater than the version that's already there.  Start at `1.0.0`

Reply:
Thanks, I have no idea what I'm doing here. This whole file is just a wild guess.